contents/haskell/Haskell.md

```diff
@@ -0,0 +1,195 @@
+# Introduction to Haskell
+
+Authors: Thenaesh Elango
+
+
+# Overview
+
+Haskell is a purely functional programming language with strong, static,
+inferred typing. It is one of the languages that feature the
+[second-order lambda calculus](https://en.wikipedia.org/wiki/System_F)
+(the other being the ML family of languages), which lends the language much of
+its expressive power.
+
+While Haskell has its roots in academia, its emphasis on purity and
+side-effect-free computation makes it a valuable asset in software engineering
+contexts. Programs written in Haskell tend to be easy to test, refactor and
+debug, with the compiler usually catching all bugs before the program can even
+be compiled and run. Consequently, Haskell codebases are extraordinarily stable.
```

Comment:
Well, no concrete examples (I don't think people go around actually documenting occurrences of this), but lots of pages explaining this. I'll throw one in here.